src/main/java/frc/robot/SwerveModule.java

```diff
@@ -190,40 +187,40 @@ public double getRotPosition() {
         return getRotPositionRaw();
     }
 
-    public double getRotRelativePosition(){
-        return _rotRelativeEncoder.getPosition()/12.8;
+    public double getRotRelativePosition() {
+        return mRotRelativeEncoder.getPosition() / 12.8;
```

Comment:
Either create a named constant for 12.8 in this class or in SwerveConstants.java

Reply:
Done